astropy/units/tests/test_structured.py

```diff
@@ -5,12 +5,15 @@
 """
 import pytest
 import numpy as np
+import numpy.lib.recfunctions as rfn
 from numpy.testing import assert_array_equal
 
 from astropy import units as u
 from astropy.units import StructuredUnit, Unit, UnitBase, Quantity
 from astropy.utils.masked import Masked
 
+from .test_quantity_non_ufuncs import CoverageMeta
```

Comment:
One disadvantage of putting the tests here is that if one now tests just `pytest test_quantity_non_ufuncs.py`, one will get a failure. My sense would be to move the tests, especially since the actual tests do not really use the class setup much (you could still keep that test for failure here).

Reply:
✅

Reply:
Still to remove the import!

Reply:
Ah. IMO that would be nice as pre-commit hook : to check for unused imports and error unless #noqa

Reply:
Yes, along with other PEP stuff...